src/hsmm_mvpy/models.py

```diff
@@ -966,7 +966,7 @@ def estim_probs(self, magnitudes, parameters, n_events=None, subset_epochs=None,
             backward = np.zeros((self.max_d, n_trials, n_events), dtype=np.float64)
             # Computing forward and backward helper variable
             #  when stage = 0:
-            forward[:,:,0] = np.tile(pmf[:,0][np.newaxis].T,\
+            forward[:,:,0] = np.tile(np.concatenate([[0],pmf[:-1,0]])[np.newaxis].T,\
```

Comment:
We agreed on this but somehow this is a strange fix, I'll investigate after the merge